1.Core/UIElements/Drivers/GenericWebDriver.cs

```diff
@@ -23,4 +21,12 @@ public static void Dispose()
             _driver = null;
         }
     }
+
+    public static IWebDriver LoadDriver()
+    {
+        var key = ConfigBuilder.Instance.GetString("DriverLocation");
```

Comment:
This key should be added on the json config file so the local its always a default

Reply:
Done, thank you!